src/data/faq_de.json

```
@@ -376,6 +376,217 @@
+                                ]
+                            },
+                            {
+                                "title": "Muss ich ein Foto meines Tests machen, um den Test nachzuweisen?",
+                                "anchor": "warn_without_tan_photo",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, in der Corona-Warn-App ist nicht vorgesehen, Fotos oder andere Dokumente zum Nachweis, dass ein Test wirklich stattgefunden hat, hochzuladen oder anderweitig zu prozessieren. Wir setzen hier bewusst auf die Umsicht und Eigenverantwortung unserer Anwender*innen."
+                                ]
+                            },
+                            {
+                                "title": "Wie oft kann ich warnen?",
+                                "anchor": "warn_without_tan_how_often",
+                                "active": false,
+                                "textblock": [
+                                    "Innerhalb einer bestimmten Periode können Sie einmal ohne TAN warnen. Nach Ablauf dieser Sperrfrist können Sie erneut warnen. Grundsätzlich gibt es nach oben keine Grenzen, wie oft Sie warnen können, solange diese Abstände nicht unterschritten werden. Wir hoffen aber natürlich, dass es keine Notwendigkeit für Sie gibt, in noch kürzeren Abständen immer wieder neu zu warnen."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich in Vertretung von Familienmitgliedern warnen, z.B. für mein infiziertes Kind?",
+                                "anchor": "warn_without_tan_family",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, das ist technisch nicht möglich, weil die Risikokontakte der positiv getesteten Person nicht zwingend durch das Contact Tracing auch auf ihrem Smartphone aufgezeichnet wurden. Eine stellvertretende Warnung von Ihrem Smartphone aus würde daher diese Kontakte gar nicht erreichen, sondern nur Ihre (also die falschen) Kontakte."
+                                ]
+                            },
+                            {
+                                "title": "Ich war in einem Restaurant (auf einem Event) per QR-Code eingecheckt. Kann ich die Gäste des Events mit der neuen Funktion warnen? Bleibe ich dabei anonym?",
+                                "anchor": "warn_without_tan_event",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, wenn Sie sich per Event-QR-Code eingecheckt haben, können Sie den Zeitraum Ihres Check-ins die anderen Gäste weiterhin warnen. Dabei bleibt, wie bisher, Ihre Identität anonym."

```

This is grammatically incorrect. I fixed it here:

```suggestion
                                    "Ja, wenn Sie sich per Event-QR-Code eingecheckt haben, können Sie in dem Zeitraum Ihres Check-ins die anderen Gäste weiterhin warnen. Dabei bleibt, wie bisher, Ihre Identität anonym."
```

---

Your suggestion has been comitted, thanks